helm/config_test.go

```diff
@@ -645,20 +645,14 @@ Mapping:
 }
 
 func TestHelmMultiLineScalar(t *testing.T) {
-	root := NewNodeMapping("Scalar", NewScalar("foo\nbar\nbaz"))
-	list := NewNodeList(NewScalar("one\ntwo\nthree"))
+	root := NewNodeMapping("Scalar", NewScalar("\"foo\nbar\nbaz\""))
+	list := NewNodeList(NewScalar("\"one\ntwo\nthree\""))
 	root.AddNode("List", list)
 
 	expect := `---
-Scalar: |-
-    foo
-    bar
-    baz
+Scalar: "foo\nbar\nbaz"
```

Comment:
Ah, because the `equal()` function checks string equality, not YAML-serialization equality.